src/mappings/referrerRewardHandler.ts

```diff
@@ -5,9 +5,10 @@ import { formatU128ToBalance } from "./utils";
 export async function referrerRewardHandler(
 	event: SubstrateEvent
 ): Promise<void> {
+
 	const {
 		event: {
-			data: [referree, referrer, amount],
+			data: [referral, referrer, amount],
```

Comment:
Seems like we have a definition error in https://soramitsu.atlassian.net/browse/PW-404